tools/lsp/language/completion.rs

```diff
@@ -458,6 +459,29 @@ fn resolve_element_scope(
     Some(result)
 }
 
+/// Given a property name in the specified element, gibve the non-normalized name (so that the '_' and '-' fits the definition of the property)
```

Comment:
```suggestion
/// Given a property name in the specified element, give the non-normalized name (so that the '_' and '-' fits the definition of the property)
```